Makefile

```diff
@@ -44,7 +44,7 @@ endif
 IMAGE_PLATFORMS ?= linux/amd64,linux/arm64
 
 # Base build image to use.
-BUILD_BASE_IMAGE ?= golang:1.22.5@sha256:829eff99a4b2abffe68f6a3847337bf6455d69d17e49ec1a97dac78834754bd6
+BUILD_BASE_IMAGE ?= golang:1.22.8@sha256:9f7fab27a7199abd0902daf34abf1e93aca4bafda4228bd303fdd7b4cbfab082
```

Comment:
nit: just curious where this digest came from? seeing a different one here: https://hub.docker.com/layers/library/golang/1.22.8/images/sha256-9e7db50b9858e9cd804043200f1e6acd5a11111151ce886951c9fe3523002cea?context=explore

`sha256:0ca97f4ab335f4b284a5b8190980c7cdc21d320d529f2b643e8a8733a69bfb6b`

Reply:
I used the same page (or at least intended to), but you’re right. I will make the update. It seems they might be pushing updates and moving the tags but since Dockerhub does not log the history of previous pushes, I cannot confirm.

I worked on this around midday of October 19, and the image page shows "Last pushed 3 days ago" and when hovering over "Oct 19, 2024 at 2:22 pm," although it doesn’t mention the timezone. When checking the `Makefile` for the previous Go version 1.22.5, we have the hash
`sha256:829eff99a4b2abffe68f6a3847337bf6455d69d17e49ec1a97dac78834754bd6`
but now the [1.22.5 page](https://hub.docker.com/layers/library/golang/1.22.5/images/sha256-a07daa84ec6c320c62f1fddb40b84698de004aac1c5ceb28243d674952d536f3?context=explore) shows a different hash `sha256:86a3c48a61915a8c62c0e1d7594730399caa3feb73655dfe96c7bc17710e96cf`.

Reply:
ah yeah that is an interesting/unfortunate one if the team building those images is repushing over the same tag